### ISSUE TITLE
Naming Conflict with etnbrd/grunt-codeclimate

### ISSUE THREAD

**Issue by Xiphe** [MrBoolean/grunt-codeclimate-reporter#1]
Hey it seems as if [etnbrd/grunt-codeclimate](https://github.com/etnbrd/grunt-codeclimate) does the same. 

Since you both choose the same name, I'm often confuded which plugin is installed via npm. 
It would be great if you could talk to @etnbrd in order to resolve this naming conflict.

**Comment by etnbrd** [comment]
Hi,

I am no longer maintaining this package as I don't use it currently.
I think I have the name grunt-codeclimate on npm and am willing to give it,
if the package of MrBoolean is maintained (or a little less in abandon than
mine :p ).

**Comment by Xiphe** [comment]
looking at both repos, they seem to do nearly the same. Yours @etnbrd contains tests are they still working?

**Comment by etnbrd** [comment]
There is indeed a folder named test, with a test file in it, but it is
empty.

If I remember correctly, I built this grunt task from a grunt task example
and the code-climate reporter repository (probably this one :
https://github.com/codeclimate/javascript-test-reporter).
At the time, I did a pull request to separate the code-climate reporting
library from the grunt-task, but it never merged.
https://github.com/codeclimate/javascript-test-reporter/pull/7
